### PR TITLE
Clear the last cookie queue

### PR DESCRIPTION
**Pull request by hhxsv5**

### CODE REVIEW

File: src/Base.php [hunk 1]
```diff
@@ -178,6 +182,17 @@ private function dealWithResponse($response, $illuminate_response, $accept_gzip)
...
 
+    protected function clean(IlluminateRequest $request)
+    {
+        // Clean laravel cookie queue
+        $cookies = $this->app->make(CookieJar::class);
+        foreach ($cookies->getQueuedCookies() as $name => $cookie) {
+            $cookies->unqueue($name);
+        }
+
+        //...
```
Comment (garveen) [inline src/Base.php]: should cleanup

File: src/Base.php [hunk 2]
```diff
@@ -178,6 +182,17 @@ private function dealWithResponse($response, $illuminate_response, $accept_gzip)
         $this->endResponse($response, $content);
     }
 
+    protected function clean(IlluminateRequest $request)
+    {
+        // Clean laravel cookie queue
+        $cookies = $this->app->make(CookieJar::class);
```
Comment (garveen) [inline src/Base.php]: Is there any way to know the request have a queue before do this heavy thing?

File: src/Base.php [hunk 3]
```diff
@@ -178,6 +182,17 @@ private function dealWithResponse($response, $illuminate_response, $accept_gzip)
         $this->endResponse($response, $content);
     }
 
+    protected function clean(IlluminateRequest $request)
```
Comment (garveen) [inline src/Base.php:177]: It looks like the function did not use `$request`?

**Comment by garveen** [comment]
The clear job should at the same location as others like https://github.com/garveen/laravoole/blob/master/src/Base.php#L117

File: src/Base.php (hunk 2 above)
Comment (hhxsv5) [reply src/Base.php]: As far as I know, this the only way.

**Comment by hhxsv5** [comment]
OK

File: src/Base.php (hunk 3 above)
Comment (hhxsv5) [reply src/Base.php:177]: Yeah, long-term consideration

**Comment by hhxsv5** [comment]
Please review.

**Comment by hhxsv5** [comment]
Any news?

**Comment by hhxsv5** [comment]
Thanks, could you please make a release?

**Pull request merged**